### DISCUSSION TITLE
Ability to turn off sign notifications?

### DISCUSSION THREAD

**Discussion by dark-swordsman** [GriefPrevention/GriefPrevention#883] [Q&A] [answered]
Hello,

I noticed on our server, by default, it will say "x placed a sign @..." and show what's on the board. 

I am not super versed with Minecraft servers, but is there a specific reason/benefit to having this? Is it something we can disable?

Tried looking in the Wiki, issues, and other parts of google but I couldn't find any reason.

**Comment by bigpresh** [comment, marked as answer]
From the wiki:

> Sometimes, griefers will place signs to communicate secretly or anonymously. For this reason, all sign placements are logged so that you may investigate any issues.


It's logged because it can be useful, but if you don't find it useful, there's a  `AdminsGetSignNotifications: true` setting which should allow you to disable it.

**Comment by dark-swordsman** [comment]
Okay, thanks!
